parastell.py

```diff
@@ -202,15 +204,80 @@ def native_export():
+        # get a tuple of volume ids, these will be in the import order
+        # and are assumed to increase in order from inside to outside
+        volumes = cubit.get_entities('volume')
+
+        # tracks the surface id of the outer surface of the previous layer
+        lastOuterSurface = None
+        
+        for name, vol in zip(components.keys(),volumes):
+
+            # wait to merge until the next layer if the plasma is included
+            # store surface to be merged for next loop
+            if name == 'plasma':
+                surfaces = cubit.get_relatives('volume',vol,'surface')
+
+                for surf in surfaces:
+                    if cubit.get_surface_type(surf) == 'spline surface':
+                        lastOuterSurface = surf
+
+            # check if we are in the first layer in a build with plasma excluded
+            # store outer surface to be merged in next loop
+            elif lastOuterSurface is None:
+                surfaces = cubit.get_relatives('volume',vol,'surface')
+                splineSurfaces = []
+
+                for surf in surfaces:
+                    if cubit.get_surface_type(surf) == 'spline surface':
+                        splineSurfaces.append(surf)
+
+                if cubit.get_bounding_box('surface', splineSurfaces[1])[4] > cubit.get_bounding_box('surface',splineSurfaces[0])[4]:
+                    outerSurface = splineSurfaces[1]
+
+                else:
+                    outerSurface = splineSurfaces[0]
+
+                lastOuterSurface = outerSurface
+
+            # get the inner surface of the current if it is not the first
+            # layer and merge it with the outside surface of the previous layer
+            else:
+                surfaces = cubit.get_relatives('volume',vol,'surface')
+                splineSurfaces = []
+
+                for surf in surfaces:
+                    if cubit.get_surface_type(surf) == 'spline surface':
+                        splineSurfaces.append(surf)
+
+                #the outer surface bounding box will have the larger max xy value
+                if cubit.get_bounding_box('surface', splineSurfaces[1])[4] > cubit.get_bounding_box('surface',splineSurfaces[0])[4]:
+                    outerSurface = splineSurfaces[1]
+                    innerSurface = splineSurfaces[0]
+
+                else:
+                    
+                    outerSurface = splineSurfaces[0]
+                    innerSurface = splineSurfaces[1]
+
+                cubit.cmd('merge surface '+ str(innerSurface) + ' ' + str(lastOuterSurface))
+                
+                lastOuterSurface = outerSurface
 
     for name in components.keys():
         import_path = dir / Path(name).with_suffix('.step')
         cubit.cmd(f'import step "{import_path}" heal')
         components[name]['vol_id'] = cubit.get_last_id("volume")
 
-    cubit.cmd('imprint volume all')
-    cubit.cmd('merge volume all')
-
+    merge_layer_surfaces()
+            
```

Comment:
This should probably be an option that is triggered by some variable.

Reply:
added it as an option in the `export` dict, updated example script. this will default to false.